src/Ninja/G1CP/Content/Tests/test127.d

```diff
@@ -0,0 +1,21 @@
+/*
+ * #127 One chest in NC cannot be opened
+ *
+ * There does not seem an easy way to test this fix programmatically, so this test relies on manual confirmation.
+ *
+ * Expected behavior: The chest can be picked with the combination "LRL".
+ */
+func void G1CP_Test_127() {
+    G1CP_Testsuite_CheckManual();
+    var zCWaypoint wp; wp = G1CP_Testsuite_FindWaypoint("NC_HUT28_OUT");
+    var int itemId; itemId = G1CP_Testsuite_CheckItem("ITKELOCKPICK");
+    G1CP_Testsuite_CheckPassed();
+
+    // Give the player an item to check
+    if (!Npc_HasItems(hero, itemId)) {
+        CreateInvItem(hero, itemId);
+    };
+
```

Comment:
What do you think about adding a screen print to show the lock picking combination (e.g. simply with `Print("The lock picking combination is LRL")`)? That may be a helpful enhancement for running the test.